### ISSUE TITLE
Really like the chess part of this project!

### ISSUE THREAD

**Issue by kenpchess** [nguyenpham/FelicityEgtb#3]
I am really enjoying the classic chess side of this project pham! I generated many of the 5-man tables also (on a 3.6ghz quad-core mac-mini no less!) just for fun. Took quite a while but enjoyed learning about the order for the generating process. 

I hope you keep the chess side of this project going! I hope six-men and maybe speed-ups are next?

Thanks again and keep up the good chess research work!

 - kenpchess

**Comment by nguyenpham** [comment]
Yes, 6-men for chess will be next, after checking and speeding up
